src/sentry/api/serializers/models/sentry_app_component.py

```diff
@@ -13,6 +14,7 @@ def serialize(self, obj, attrs, user):
                 "uuid": obj.sentry_app.uuid,
                 "slug": obj.sentry_app.slug,
                 "name": obj.sentry_app.name,
+                "avatars": get_sentry_app_avatars(sentry_app=obj.sentry_app),
```

Comment:
Need access to the avatars to avoid making another API request on the Issue Details page.